content/docs/control_plane_health_probes.md

```diff
@@ -123,8 +123,8 @@ If any health probes are consistently failing, perform the following steps to id
     $ osm mesh list
 
     MESH NAME   NAMESPACE      CONTROLLER PODS                  VERSION     SMI SUPPORTED
-    osm         osm-system     osm-controller-5494bcffb6-qpjdv  v0.8.4      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
-    osm2        osm-system-2   osm-controller-48fd3c810d-sornc  v0.8.4      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
+    osm         osm-system     osm-controller-5494bcffb6-qpjdv  v0.9.0      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
+    osm2        osm-system-2   osm-controller-48fd3c810d-sornc  v0.9.0      TrafficSplit:split.smi-spec.io/v1alpha2,TrafficTarget:access.smi-spec.io/v1alpha3,HTTPRouteGroup:specs.smi-spec.io/v1alpha4,TCPRoute:specs.smi-spec.io/v1alpha4
```

Comment:
Might be a good idea to check and see if the SMI SUPPORTED items have any particular sort order and if so, did the default change; I tested with the v0.9.0 RC and got a different order (HTTPRouteGroup was first).

Reply:
thank you @bridgetkromhout 

fixed